src/benchmarking/run_benchmarking.sh

```diff
@@ -51,8 +51,9 @@ function test_kraken2() {
                basefile="$(basename -- $file)";
                test_dir_name="$(basename "${test_dir##*/}")";
                mkdir -p results/${tool_name}/${test_dir_name}/;
+               mkdir -p results/${tool_name}/${test_dir_name}/${basefile%${file_suffix}}/;
                kraken2 --use-names --threads 4 --db tools/${tool_name}/minikraken2_v2_8GB_201904_UPDATE \
-                        --report results/${tool_name}/${test_dir_name}/${basefile%${file_suffix}}.report \
+                        --report results/${tool_name}/${test_dir_name}/${basefile%${file_suffix}}/binning.report \
```

Comment:
Got it! I like this, that way we can use the `binning.report`. I think talked about this yesterday